dev/Generated/Pager.properties.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// DO NOT EDIT! This file was generated by CustomTasks.DependencyPropertyCodeGen
+#include "pch.h"
+#include "common.h"
+#include "Pager.h"
+
+namespace winrt::Microsoft::UI::Xaml::Controls
+{
+    CppWinRTActivatableClassWithDPFactory(Pager)
+}
+
+#include "Pager.g.cpp"
+
+GlobalDependencyProperty PagerProperties::s_PlaceholderProperty{ nullptr };
+
+PagerProperties::PagerProperties()
+{
```

Comment:
Is there a pager.idl file that is generating the properties.cpp/.h files ? I was not expecting to see these files. If these are unused, you can remove them.

Reply:
Thanks for catching this. I will delete this file immediately